Sprint-3/slideshow/slideshow.js

```diff
@@ -1,8 +1,50 @@
 const images = [
-    "./assets/cute-cat-a.png",
-    "./assets/cute-cat-b.jpg",
-    "./assets/cute-cat-c.jpg",
+  "./assets/dog1.webp",
+  "./assets/dog2.webp",
+  "./assets/dog3.avif",
+  "./assets/dog5.webp",
 ];
+let i = 0;
```

Comment:
`i` is widely used as a loop variable; it is dangerous to use this identifier globally.

Reply:
Dear @cjyuan , that's a very helpful comment, but I can't find a proper place in my codes to put it in, unless I change my codes completely. could you recommend me how to do that, please.
Thanks in advance

Reply:
Use another name for this variable?